app/src/renderer/components/network/PageNetwork.vue

```diff
@@ -40,15 +40,23 @@
             <dl class="info_dl">
               <dt>Total Liquid {{ bondDenom }}</dt>
               <dd id="loose_tokens">
-                {{ pool.pool.loose_tokens ? pool.pool.loose_tokens : `n/a` }}
+                {{ pool.pool.not_bonded_tokens ? num.pretty(pool.pool.not_bonded_tokens) : `n/a` }}
```

Comment:
Line 43 and 51 share the same logic (and 59 is partially doing that via `outstandingRewards`), can be moved to a function that takes as argument an integer

Reply:
> can be moved to a function that takes as argument an integer

I like this idea, I'd add also a string parameter for the default response too (e.g: `n/a`). We could move this into a helper function in `scripts` since we use it in more pages as well